### PR TITLE
Modify example JSON to CSV to use two instances.

### LINKED ISSUE #71: More complete example of using process_json_to_csv.jl
It's not clear from the current README of how to use process_json_to_csv.jl to process a large collection of solves like we do in the paper. To make this usage clearer, provide a small example where we enumerate a few configurations over few instances and use process_json_to_csv.jl to process the results.

### PR DESCRIPTION
**Pull request by wschudy**
Fixes #71. Also replaces names with configs, which was leftover from pull request #47.

### CODE REVIEW

File: benchmarking/README.md [hunk 1]
```diff
@@ -101,15 +101,22 @@ into a CSV file for analysis.
...
---output_dir /tmp/first_order_lp_solve
-$ julia --project=scripts scripts/solve_lp_external.jl \
---instance_path test/trivial_lp_model.mps --solver scs-indirect \
---tolerance 1e-7 --output_dir /tmp/scs_solve
+$ benchmarking/collect_mittelmann_benchmark.sh /tmp ~/mittelmann_benchmark
+$ for INSTANCE in nug08-3rd qap15
+do
+  julia --project=scripts scripts/solve_qp.jl \
+    --instance_path ~/mittelmann_benchmark/$INSTANCE.mps.gz --method pdhg \
+    --output_dir /tmp/first_order_lp_solve_$INSTANCE
```
Comment (mlubin) [inline benchmarking/README.md:109]: The output dir doesn't need to vary with the instance name. The scripts allow results from multiple instance to be in the same directory. Instead, the directory should vary with the solver configuration, e.g., solve tolerance.

File: benchmarking/README.md [hunk 2]
```diff
@@ -101,15 +101,22 @@ into a CSV file for analysis.
...
---instance_path test/trivial_lp_model.mps --method pdhg \
---output_dir /tmp/first_order_lp_solve
-$ julia --project=scripts scripts/solve_lp_external.jl \
---instance_path test/trivial_lp_model.mps --solver scs-indirect \
---tolerance 1e-7 --output_dir /tmp/scs_solve
+$ benchmarking/collect_mittelmann_benchmark.sh /tmp ~/mittelmann_benchmark
+$ for INSTANCE in nug08-3rd qap15
+do
+  julia --project=scripts scripts/solve_qp.jl \
+    --instance_path ~/mittelmann_benchmark/$INSTANCE.mps.gz --method pdhg \
```
Comment (mlubin) [inline benchmarking/README.md:108]: Nit: We use google bash style, i.e., ${INSTANCE}.mps.gz

File: benchmarking/README.md (hunk 1 above)
Comment (wschudy) [reply benchmarking/README.md]: Done.

File: benchmarking/README.md (hunk 2 above)
Comment (wschudy) [reply benchmarking/README.md]: Done.

**Comment by wschudy** [comment]
PTAL. I also replaced names with configs, which was leftover from pull request #47.

File: benchmarking/README.md [hunk 3]
```diff
@@ -101,15 +101,20 @@ into a CSV file for analysis.
 For example (from the `FirstOrderLp.jl` root directory),
 
 ```sh
-$ julia --project=scripts scripts/solve_qp.jl \
---instance_path test/trivial_lp_model.mps --method pdhg \
---output_dir /tmp/first_order_lp_solve
-$ julia --project=scripts scripts/solve_lp_external.jl \
---instance_path test/trivial_lp_model.mps --solver scs-indirect \
---tolerance 1e-7 --output_dir /tmp/scs_solve
+$ benchmarking/collect_mittelmann_benchmark.sh /tmp ~/mittelmann_benchmark
```
[the remark this answers is not in the document]
Comment (wschudy) [reply benchmarking/README.md]: Done.

File: benchmarking/README.md [hunk 4]
```diff
@@ -101,15 +101,20 @@ into a CSV file for analysis.
...
+$ for INSTANCE in nug08-3rd qap15
+do
+  julia --project=scripts scripts/solve_qp.jl \
+    --instance_path ~/mittelmann_benchmark/${INSTANCE}.mps.gz --method pdhg \
+    --output_dir /tmp/first_order_lp_solve
+  julia --project=scripts scripts/solve_lp_external.jl \
+    --instance_path ~/mittelmann_benchmark/${INSTANCE}.mps.gz --solver scs-indirect \
+    --tolerance 1e-7 --output_dir /tmp/scs_solve
```
[the remark this answers is not in the document]
Comment (wschudy) [reply benchmarking/README.md]: Done.

File: benchmarking/README.md [hunk 5]
```diff
@@ -101,15 +101,21 @@ into a CSV file for analysis.
...
---instance_path test/trivial_lp_model.mps --method pdhg \
---output_dir /tmp/first_order_lp_solve
-$ julia --project=scripts scripts/solve_lp_external.jl \
---instance_path test/trivial_lp_model.mps --solver scs-indirect \
---tolerance 1e-7 --output_dir /tmp/scs_solve
+$ benchmarking/collect_lp_benchmark.sh /tmp ~/lp_benchmark
+$ for INSTANCE in nug08-3rd qap15
+do
+  julia --project=scripts scripts/solve_qp.jl \
+    --instance_path ~/lp_benchmark/${INSTANCE}.mps.gz --method pdhg \
```
Comment (mlubin) [inline benchmarking/README.md:108]: #78 just refactored the instructions to use $HOME everywhere instead of ~

Comment (wschudy) [reply benchmarking/README.md]: Done.

**Review by mlubin — APPROVED** [review]

**Pull request merged**